data/json/recipes/armor/pets_cow.json

```diff
@@ -109,7 +109,7 @@
     "components": [
       [ [ "water", 36 ], [ "water_clean", 36 ] ],
       [
-        [ "wax", 7 ],
+        [ "wax_any", 7, "LIST" ],
```

Comment:
So if I'm understanding you correctly, you need to get rid of the copy-from in this and two other recipes and explicitly define everything and that will make this PR green?

Reply:
I think the copy-from is the problem caused, yeah. But also I have no idea how copy-from works with recipes, so I could not tell you what exactly would need to be done. I technically could probably do that but I've opted not to in hope that the underlying issue would be fixed... But it's been almost a year so maybe it's time to admit defeat...